apps/google-analytics-4/lambda/package.json

```diff
@@ -11,7 +11,8 @@
     "start:dev": "nodemon",
     "test": "TS_NODE_TRANSPILE_ONLY=1 mocha",
     "test:debug": "TS_NODE_TRANSPILE_ONLY=1 mocha -- --inspect --inspect-brk",
-    "test:watch": "TS_NODE_TRANSPILE_ONLY=1 mocha --watch --watch-files src --watch-files test"
+    "test:watch": "TS_NODE_TRANSPILE_ONLY=1 mocha --watch --watch-files src --watch-files test",
+    "deploy:test": "SLS_DEBUG=* STAGE=test sls deploy"
```

Comment:
only `deploy:test` for now, since lerna will pick up the deploy automatically if we leave a deploy command in